server/views/case-list.njk

```diff
@@ -3,7 +3,7 @@
 {%- from "components/tabs/macro.njk" import pacTabs -%}
 
 {% block javascripts %}
-    <script type="text/javascript" src="/javascripts/filter-component-{{ appVersion }}.min.js"></script>
+    <script type="text/javascript" src="{{components.filterComponent.javascript}}"></script>
```

Comment:
Is this a new way to get the file path? Not seen it and it looks cool?

Reply:
Ive moved the file path into the "nunchucksComponents" utils, the idea being to componentise the filter so we don't have to make changes in 4 places every time we update it